docker-compose.yml

```diff
@@ -8,7 +8,7 @@ volumes:
 services:
   db:
     ports:
-      - "3306:3306"
+      - "3308:3308"
```

Comment:
This change is still appearing in the PR.

Reply:
Mmm..default mysql port changes it somehow , sorry once again for the same